spec/models/organization_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "spec_helper"
+
+describe Organization do
+  it "1:  without http & https" do
+    org = Organization.new
+    org.web = "www.casa.com"
+    org.ensure_url
+    expect(org.web).to eq "http://www.casa.com"
```

Comment:
Añade `expect(org).to be_valid` o `expect(org).not_to be_valid` a todos los ejemplos. Como habrán otras razones para que se invalide, en un `before` puedes ya construir una org con datos buenos por ejemplo
```ruby
describe Organization do
  let(:org) { Organization.new(.....datos para que sea válida, exceptuando la url....) }
  # por ejemplo let(:org) { Organization.new(name: "Banc del Temps dels Quatre Pins") }

  it "1:  without http & https" do
    # org ya lo tienes disponible por el `let` de arriba
    org.web = "www.casa.com"
    expect(org).to be_valid
    expect(org.web).to eq "http://www.casa.com"
  end
end
```
es importante que controles antes la validez, porque así le das el tiempo de ejecutarse.